docs/core/plugins/sieve.md

```diff
@@ -578,12 +641,9 @@ the user's home directory.
 The Dovecot Sieve plugin allows executing multiple Sieve scripts
 sequentially. The extra scripts can be executed before and after the
 user's private script. For example, this allows executing global Sieve
-policies before the user's script. This is not possible using the
-[[setting,sieve_default]] setting because that is only used when the
-user's private script does not exist.
-
-See [[setting,sieve_before]] and [[setting,sieve_after]] for details on how
-to configure the execution sequence.
+policies before the user's script. See the `before` and `after` Sieve storage
+types [[link,sieve_storage_type]] for details on how to configure the execution
```

Comment:
* `[[link,sieve_storage_type_before,before]] and [[link,sieve_storage_type_after,after]]`
 * I guess remove `[[link,sieve_storage_type]]`

Reply:
fixed